### PR TITLE
Support zellowork by .Net example

### PR DESCRIPTION
**Pull request by ihorserba**

### CODE REVIEW

File: examples/cs/ZelloMediaStream.cs [hunk 1]
```diff
@@ -151,21 +158,30 @@ public bool Connect()
...
+            dynamic json = new ExpandoObject();
+
+            json.seq = 1;
+            json.command = "logon";
+            json.username = this.Configuration["zello:username"];
+            json.password = this.Configuration["zello:password"];
+            json.auth_token = this.Configuration["zello:token"];
+            json.channel = this.Configuration["zello:channel"];
+
+            if (this.Configuration.GetSection("zello:network").Exists())
```
Comment (vocoded) [inline examples/cs/ZelloMediaStream.cs:172]: can remove

File: examples/cs/ZelloMediaStream.cs [hunk 2]
```diff
@@ -151,21 +158,30 @@ public bool Connect()
             return false;
         }
 
+        private dynamic GetLogonJson()
+        {
+            dynamic json = new ExpandoObject();
```
Comment (vocoded) [inline examples/cs/ZelloMediaStream.cs:163]: can we create the initial object here?

```
dynamic json = {
  seq = 1,
...
```

File: examples/cs/ZelloMediaStream.cs [hunk 3]
```diff
@@ -12,15 +13,18 @@
 
     class ZelloMediaStream : IDisposable
     {
+        const string defaultServerUrlWork = "wss://zellowork.io/ws/";
+        const string defaultServerUrlConsumer = "wss://zello.io/ws";
```
Comment (vocoded) [inline examples/cs/ZelloMediaStream.cs:17]: let's remove "default" from these names

File: examples/cs/ZelloMediaStream.cs (hunk 1 above)
Comment (ihorserba) [reply examples/cs/ZelloMediaStream.cs]: Thanks, not needed indeed

File: examples/cs/ZelloMediaStream.cs (hunk 2 above)
Comment (ihorserba) [reply examples/cs/ZelloMediaStream.cs]: reverted this change

File: examples/cs/ZelloMediaStream.cs (hunk 3 above)
Comment (ihorserba) [reply examples/cs/ZelloMediaStream.cs]: removed

**Comment by ihorserba** [comment]
Changed the auth token to be optional.

**Pull request merged**